tdp/core/deployment/test_deployment_runner.py

```diff
@@ -204,9 +207,12 @@ def test_service_log_not_emitted_when_config_start_wrong_order(
     )
     deployment_iterator = mock_deployment_runner.run(deployment)
 
-    for _ in deployment_iterator:
-        if _:
-            _()
+    for (
+        op,
+        fn,
+    ) in deployment_iterator:
```

Comment:
You can not one line this like other ? This is enforce by the linter ?

Reply:
No it was not enforeced, it has been written on one line